src/condor_scripts/make-personal-from-tarball

```diff
@@ -33,6 +45,11 @@ if [[ -e ${LOCAL_CONFIG_DIR}/00-personal-condor ]]; then
     exit 1
 fi
 
+if [[ -e ${LOCAL_CONFIG_DIR}/00-ap-condor ]]; then
+    echo "Refusing to overwrite existing local/config.d/00-ap-condor file, aborting."
+    exit 1
+fi
```

Comment:
I don't see any code that creates 00-ap-condor. This script always creates 00-personal-condor.